crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -248,6 +250,13 @@ impl ServerBuilder {
         let registry_service = mysten_metrics::start_prometheus_server(prom_addr);
         info!("Starting Prometheus HTTP endpoint at {}", prom_addr);
         let registry = registry_service.default_registry();
+        registry
+            .register(mysten_metrics::uptime_metric(
+                "graphql",
+                version.version,
+                "unknown",
```

Comment:
This parameter is supposed to be the chain identifier right? Would it be useful if we also include the chain identifier here?

Reply:
It's a nice idea, but I think it might be difficult to implement it, because the GraphQL service would need to talk to its DB to get the chain identifier, and so requiring that during the set-up may get complicated? Not sure, I just get nervous about the idea of adding too much logic before the metrics come up, because we're blind at that point.

Reply:
@emmazzz Ideally, yes, we should have this. 
The way the `uptime_metric` was designed was mostly for sui-node and sui-proxy, where somehow is easy to get the chain identifier from its state. As @amnn mentioned, we'd have to either ask the DB about it, or update this metric later, once the service started and is running.
The cleanest way would be for this argument to be an `Option` instead of a `&str`, so in cases like ours we can skip it.
Given how we set up our dashboard, with the possibility of selecting the desired network, I would think this is probably ok.